bugRzilla_review.Rmd

```diff
@@ -465,46 +456,73 @@ datatable(head(baa, 5), options = list(scrollX = TRUE))
 
 ### Visualizations
 
+```{r}
+#Counting number of bugs per month in an year
+bugs_counts <- baa %>%
+    arrange(bug_id) %>%
+    count(creation_year)
+skim(head(bugs_counts))
```

Comment:
This only tell us about the first 5 rows not about the variables on the whole table